tools/eslint-rules/lowercase-name-for-primitive.js

```diff
@@ -22,11 +22,11 @@ module.exports = function(context) {
 
     switch (names.type) {
       case 'Literal':
-        checkName(node, names.value);
+        checkName(names, names.value);
         break;
       case 'ArrayExpression':
-        names.elements.forEach((name) => {
-          checkName(node, name.value);
+        names.elements.forEach((name, index) => {
+          checkName(names.elements[index], name.value);
```

Comment:
Couldn't this just be `name` instead of `names.elements[index]`? Pretty sure we can also get rid of the second argument here and above.

Reply:
👍🏻